package.json

```diff
@@ -105,7 +105,7 @@
     "playwright": "^1.23.4",
     "portfinder": "^1.0.28",
     "puppeteer": "^15.4.0",
-    "super-simple-web-server": "^1.1.2",
+    "super-simple-web-server": "garris/superSimpleExpressServer#b6b9684514caa7b95d0920be2d56946e5e636693",
```

Comment:
Note that this dependency is currently pointing at this PR, because it allows the sort to the ssws to be configured: https://github.com/garris/superSimpleExpressServer/pull/4
Once that PR is merged, the code in this repo should point back at the next stable release that contains the PR.